src/style/footer.scss

```
@@ -0,0 +1,145 @@
+.footer {
+    margin-top: 65px;
+    background-color: $Black;
+    color: $white;
+    font-family: $Roboto;
+
+    display:flex;
+    flex-direction: column;
+}
+.footer__first {
+    display: flex;
+    gap: 175px;
+}
+.footer__first__wrapper{
+    @include wrapperForFotter;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    padding-bottom: 50px;
+    padding-top: 90px;
+}
+.footer__first__h {
+    div{
+        font-family: $Forum;
+        font-size: 28px;
+    }
+    display: flex;
+    gap: 25px;
+}
+.footer__first__nav1 {
+    display: flex;
+    flex-direction: column;
+    gap: 36px;
+}
+.footer__first__nav2 {
+    display: flex;
+    flex-direction: column;
+    gap: 36px;
+}
+
+.footer__first__icons {
+    display: flex;
+    align-items: flex-end;
+
+    gap: 24px;
+    div{
+        width: 50px;
+        height: 50px;
+        border: 2px solid $white;
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+}
+.footer__line {
+    width: 100%;
+    height: 3px;
+    background-color: #666666;
+}
+.footer__second {
+    @include wrapper;
+    padding-top: 50px;
+    padding-bottom: 50px;
+
+    display: flex;
+    justify-content: space-between;
+}
+
+@media(max-width: 1131px){
+    .footer__first__nav2{
+        padding-left: 50px;
+    }
+    .footer__first{
+        gap: 40px;
+    }
+}
+@media(max-width: 919px){
+    .footer__first__nav2{
+        padding-left: 0;
+    }
+}
+@media(max-width: 827px){
+    .footer__first__wrapper{
+        align-items: center;
+        flex-direction: column;
+        justify-content: center;
+        gap:40px;
+    }
+    .footer__first{
+        gap: 90px;
+    }
+    .footer__first__nav2{
+        padding-left: 100px;
+    }
+    .footer__first__BoxIcons{
+        padding-left: 250px;
+    }
+}
+
+@media(max-width:595px){
+    .footer__first__BoxIcons {
```

Как выше писал стили пишем в kebab-case

```suggestion
    .footer__first__box-icons {
```